packages/driver-interface/src/services/amqp.ts

```diff
@@ -24,3 +20,24 @@ export const updateLocation = (
     })
     .catch(console.warn)
 }
+
+export const publishBookingEvent = (bookingId: string, event: string) =>
```

Comment:
Missing return type signature
```suggestion
export const publishBookingEvent = (bookingId: string, event: string): Promise<boolean | void> =>
```